x-pack/plugins/monitoring/public/components/cluster/overview/elasticsearch_panel.js

```diff
@@ -108,20 +136,35 @@ export function ElasticsearchPanel(props) {
                   data-test-subj="esNumberOfNodes"
                   onClick={goToNodes}
                 >
-                  Nodes: { formatNumber(get(nodes, 'count.total'), 'int_commas') }
+                  <FormattedMessage
+                    id="xpack.monitoring.cluster.overview.esPanel.nodesTotalLinkLabel"
+                    defaultMessage="Nodes: {nodesTotal}"
+                    values={{ nodesTotal: formatNumber(get(nodes, 'count.total'), 'int_commas') }}
+                  />
                 </EuiLink>
               </h3>
             </EuiTitle>
             <EuiHorizontalRule margin="m" />
             <EuiDescriptionList type="column">
-              <EuiDescriptionListTitle>Disk Available</EuiDescriptionListTitle>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.monitoring.cluster.overview.esPanel.diskAvailableLabel"
+                  defaultMessage="Disk Available"
+                />
+              </EuiDescriptionListTitle>
               <EuiDescriptionListDescription data-test-subj="esDiskAvailable">
                 <BytesUsage
                   usedBytes={get(nodes, 'fs.available_in_bytes')}
                   maxBytes={get(nodes, 'fs.total_in_bytes')}
                 />
               </EuiDescriptionListDescription>
-              <EuiDescriptionListTitle>JVM Heap</EuiDescriptionListTitle>
+              <EuiDescriptionListTitle>
+                <FormattedMessage
+                  id="xpack.monitoring.cluster.overview.esPanel.jvmHeapLabel"
+                  defaultMessage="{javaVirtualMachine} Heap"
+                  values={{ javaVirtualMachine: 'JVM' }}
```

Comment:
So `JVM` is not something translated, but I see `Elasticsearch` not used like this, does that mean `Elasticsearch` will be translated?

Reply:
No. There is also Kibana translation glossary (https://docs.google.com/document/d/1e-lHAPAtxodP8U4gbwA3lw8FyhhW7M303--77nMpe8w/edit) that contains terms that will remain untranslated.
Mostly we put in `values` such technical labels like property names [`url.body.query`](https://github.com/elastic/kibana/pull/23677/files#diff-f3fe8f44620f4738dc16f72709f37b1bR123).